UI/html/question.html

```diff
@@ -75,32 +75,33 @@ <h1>Subscribe to for news letter</h1>
             </li>
         </ul>
     </section>
+    <p id="deleteDoc"></p>
     <section id="all__blog">
         <div class="all__blog">
             <div class="item" id="single__article">
                 <!-- <div class="article"> -->
-                <h4><b>Benimana Eric</b></h4>
+                <!-- <h4><b>Benimana Eric</b></h4>
                 <p class="article__text">Prosciutto beef ribs strip steak, biltong chislic tri-tip shankle. Biltong drumstick alcatra, shankle leberkas brisket pork belly pork chop. Rump andouille pork chop, beef ribs fatback t-bone brisket shankle tur salami short ribs corned
                     beef tongue. <br>
                     <p id="like__icon"><b>bizip04@gmail.com</b>&nbsp;&nbsp;&nbsp;&nbsp;</p>
 
                     <p id="like__icon"><b>Delete</b></p>
 
-                </p>
+                </p> -->
                 <!-- </div> -->
             </div>
 
-            <div class="item" id="single__article">
-                <!-- <div class="article"> -->
-                <h4><b>Benimana Eric</b></h4>
+            <!-- <div class="item" id="single__article"> -->
+            <!-- <div class="article"> -->
+            <!-- <h4><b>Benimana Eric</b></h4>
                 <p class="article__text">Prosciutto beef ribs strip steak, biltong chislic tri-tip shankle. Biltong drumstick alcatra, shankle leberkas brisket pork belly pork chop. Rump andouille pork chop, beef ribs fatback t-bone brisket shankle tur salami short ribs corned
                     beef tongue. <br>
                     <p id="like__icon"><b>bizip04@gmail.com</b>&nbsp;&nbsp;&nbsp;</p>
                     <p id="like__icon"><b>Delete</b></p>
 
```

Comment:
Remove this too. The reason you need to remove this is because a PR is an increment to the whole product and when you deliver a feature/story you have to deliver what was asked nothing more.

Reply:
Done

Reply:
Done